app/Http/Controllers/WebhooksController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Webhook;
+use App\Repositories\Interfaces\WebhookRepositoryInterface as WebhookRepository;
+
+class WebhooksController extends Controller
+{
+    private $webhookRepository;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Http/Controllers/WebhooksController.php</br>**Line**: 13</br>Opening brace should be on a new line